### PR TITLE
more ansible runner isolated cleanup

### PR DESCRIPTION
**Pull request by ryanpetrello**
follow-up to https://github.com/ansible/awx/pull/6296

### CODE REVIEW

File: awx/main/isolated/manager.py [hunk 1]
```diff
@@ -268,13 +268,6 @@ def check(self, interval=None):
         # in the final sync
         self.consume_events()
 
-        # emit an EOF event
```
Comment (ryanpetrello) [inline awx/main/isolated/manager.py:271]: Another bit of code duplication and an opportunity to converge the standard and isolated event handling paths.

**Review by jakemcdermott — APPROVED** [review]
Screenshot from 2020-04-07 19-29-58

**Review by matburt — APPROVED** [review]

**Comment by jakemcdermott** [comment]
container group:
Screenshot from 2020-04-07 20-48-40

**Review by AlanCoding — APPROVED** [review]

**Comment by ryanpetrello** [comment]
@jakemcdermott just for posterity, the duplicate line numbers in your screenshot are this Ansible bug (soon to be addressed in 2.9):

https://github.com/ansible/ansible/pull/68520
https://github.com/ansible/awx/issues/6357

**Pull request merged**